pallets/ddc-verification/src/lib.rs

```diff
@@ -24,6 +24,7 @@ use frame_system::{
 	pallet_prelude::*,
 };
 pub use pallet::*;
+pub use pallet_staking::{self as staking};
```

Comment:
remove direct dependency on pallet_staking. add trait instead and implement needed methods. if  pallet_staking doesn't have these methods, implement another class similarly to TreasuryWrapper in runtime